source/_integrations/wake_on_lan.markdown

```diff
@@ -110,17 +110,53 @@ Here are some real-life examples of how to use the **turn_off** variable.
 
 #### Suspending Linux
 
-Suggested recipe for letting the `turn_off` script suspend a Linux computer (the **target**)
-from Home Assistant running on another Linux computer (the **server**).
-
-1. On the **server**, log in as the user account Home Assistant is running under. In this example it's `hass`.
-2. On the **server**, create SSH keys by running `ssh-keygen`. Just press enter on all questions.
-3. On the **target**, create a new account that Home Assistant can ssh into: `sudo adduser hass`. Just press enter on all questions except password. It's recommended using the same user name as on the server. If you do, you can leave out `hass@` in the SSH commands below.
-4. On the **server**, transfer your public SSH key by `ssh-copy-id hass@TARGET` where TARGET is your target machine's name or IP address. Enter the password you created in step 3.
-5. On the **server**, verify that you can reach your target machine without password by `ssh TARGET`.
-6. On the **target**, we need to let the `hass` user execute the program needed to suspend/shut down the target computer. Here is it `pm-suspend`, use `poweroff` to turn off the computer. First, get the full path: `which pm-suspend`. On my system, this is `/usr/sbin/pm-suspend`.
-7. On the **target**, using an account with sudo access (typically your main account), `sudo visudo`. Add this line last in the file: `hass ALL=NOPASSWD:/usr/sbin/pm-suspend`, where you replace `hass` with the name of your user on the target, if different, and `/usr/sbin/pm-suspend` with the command of your choice, if different.
-8. On the **server**, add the following to your configuration, replacing TARGET with the target's name:
+Recipe for the `turn_off` action to suspend a Linux **target** from Home Assistant running on a Linux **server**. Home Assistant will run a suspend command on the **target** over SSH.
+
+##### On the Target
+
+1. If needed, install OpenSSH server with package manager. For example, on Ubuntu:
+
+```sh
+sudo apt install openssh-server
+```
+
+2. Create a new system user and set password.
+
+```sh
+sudo adduser --system hass --home /var/hass --shell /bin/sh
+sudo passwd hass
+```
+
+3. Allow the user to suspend without password.
+
+**Note**: Most Linux operating systems (Ubuntu, Fedora, Arch, etc.) utilize systemd and can use the builtin `systemctl` command to manage power states. For other operating systems (Gqentoo, Void, etc.), you will need alternative methods such as `elogind` or `pm-utils`.
+
+```sh
+echo "hass ALL= NOPASSWD: /usr/bin/systemctl suspend" | sudo tee /etc/sudoers.d/hass-suspend
+```
+
+##### On the Server
```

Comment:
I think this section is now a bit odd, considering 96% of our uses Home Assistant in some sort of a container (most using Home Assistant OS, Container and Supervised installation types).

These instructions therefore won't apply to the majority of our users.